scanner/services/indexer_test.go

```diff
@@ -0,0 +1,182 @@
+package services
+
+import (
+	"context"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/quay/claircore"
+	v4 "github.com/stackrox/rox/generated/internalapi/scanner/v4"
+	"github.com/stackrox/rox/scanner/indexer/mocks"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/mock/gomock"
+)
+
+var (
+	imageURL = "https://foobar:443/image:latest"
```

Comment:
can we also test with a missing tag (so it defaults to latest) and a digest?

Reply:
This is another place you can check for how URLs look like.

Reply:
Why Scanner should accept image references without tags or digests?

Reply:
I meant to test an invalid image URL (this is tested in scanner/indexer, but not here) and another case to test a valid URL with a digest instead of tag

Reply:
This is tested here on `Test_CreateIndexReport_InvalidInput`, no?

My question to you was, why should we default no tag to latest? There isn't a point in doing it.